public/chart.css

```diff
@@ -21,3 +21,7 @@
     cursor: move;
     pointer-events: all;
 }
+.episode {
+    fill: grey;
```

Comment:
Please use HEX or RGB values. `grey` might look very different depending on the browser defaults